src/lib/Scenes/Artwork/Components/AuctionCountDownTimer.tsx

```diff
@@ -58,6 +58,7 @@ export class AuctionCountDownTimer extends React.Component<AuctionCountDownTimer
     return (
       <TimeRemaining
         labelWithTimeRemaining={sale.formattedStartDateTime}
+        labelWithoutTimeRemaining={sale.formattedStartDateTime}
```

Comment:
@lilyfromseattle And here the label wasn't appearing after the auction had ended.